doc-content/enterprise-only/installation/eap-users-create-proc.adoc

```diff
@@ -63,7 +75,32 @@ endif::[]
 If you installed {CENTRAL} and {KIE_SERVER} in the same server instance, you can create a single user that has both of these roles:
 [source,bash]
 ----
-$ ./add-user.sh -a --user <USERNAME>  --password <PASSWORD> --role admin,rest-all,kie-server
+$ ./bin/jboss-cli.sh --commands="embed-server --std-out=echo,/subsystem=elytron/filesystem-realm=ApplicationRealm:add-identity(identity=<USERNAME>),/subsystem=elytron/filesystem-realm=ApplicationRealm:set-password(identity=<USERNAME>, clear={password='<PASSWORD>'}),/subsystem=elytron/filesystem-realm=ApplicationRealm:add-identity-attribute(identity=<USERNAME>, name=role, value=['admin','rest-all','kie-server'])"
 ----
 You should install {CENTRAL} and {KIE_SERVER} on different servers in production environments.
 ====
++
+[NOTE]
```

Comment:
Make this an optional step:
Optional: To create several users at one time, create a file that contains the user data and run the following command, where <USER_DATA>.cli is the file that contains the user data:
[source,bash]
----
$ ./bin/jboss-cli.sh --file=<USER-DATA>.cli
----

Reply:
<USER_DATA>.cli should be code .